lib/resources.js

```diff
@@ -38,7 +38,49 @@ const remove = (token, path, permanently=false, success, error) =>
     }
   }, success, error);
 
+const copyOrMove = (url, token, from, path, overwrite = false, fields = '', success, error) =>
+  request.post({
+    url: url,
+    token: token,
+    query: {
+      from: from,
+      path: path,
+      overwrite: overwrite,
+      fields: fields
+    },
+  }, success, error);
+
+/**
+ * Copy File or Folder
+ * @see https://tech.yandex.com/disk/api/reference/copy-docpage/
+ * @param {string} token OAuth token
+ * @param {string} [from] File or Folder Path to copy from
+ * @param {string} [path] File or Folder Path to copy to
```

Comment:
`from` and `path` params are required, pls remove the square-brackets around them.